### PR TITLE
Change the calico manifest download task to fix the CI error on PowerVS

### PR DESCRIPTION
**Pull request by Rajalakshmi-Girish**
This change is to avoid the below error on PowerVS while the conformance jobs use the ansible playbook to create k8s cluster.

```
TASK [install-calico : Download calico manifest] *******************************
fatal: [163.68.65.236]: FAILED! => {"changed": false, "dest": "/tmp/calico.yaml", "elapsed": 10, "msg": "Request failed: <urlopen error _ssl.c:880: The handshake operation timed out>", "url": "https://docs.projectcalico.org/manifests/calico.yaml"}
```
https://github.ibm.com/powercloud/container-dev/issues/1467 has more details.
As `get_url` module of ansible doesn't have the option to limit the download rate, switching to shell and then `wget`.

https://prow.ppc64le-cloud.org/view/gs/ppc64le-kubernetes/logs/periodic-kubernetes-conformance-test-ppc64le/1390216828562706432 is the failed job.

### CODE REVIEW

**Comment by ltccci** [comment]
[APPROVALNOTIFIER] This PR is **NOT APPROVED**

This pull-request has been approved by: *<a href="https://github.com/ppc64le-cloud/k8s-ansible/pull/13#" title="Author self-approved">Rajalakshmi-Girish</a>*
To complete the [pull request process](https://git.k8s.io/community/contributors/guide/owners.md#the-code-review-process), please assign **mkumatag** after the PR has been reviewed.
You can assign the PR to them by writing `/assign @mkumatag` in a comment when ready.

The full list of commands accepted by this bot can be found [here](https://go.k8s.io/bot-commands?repo=ppc64le-cloud%2Fk8s-ansible).

<details open>
Needs approval from an approver in each of these files:

- **[OWNERS](https://github.com/ppc64le-cloud/k8s-ansible/blob/master/OWNERS)**

Approvers can indicate their approval by writing `/approve` in a comment
Approvers can cancel approval by writing `/approve cancel` in a comment
</details>

File: roles/install-calico/tasks/main.yml
Comment (sudeeshjohn) [inline roles/install-calico/tasks/main.yml:4]: is it required to clear existing `calico.yaml` if there any ?

Comment (Rajalakshmi-Girish) [reply roles/install-calico/tasks/main.yml:4]: Usually, we have been using this on a new VM hence there wasn't existing calico.yaml.

But yes it shall would be correct to change it to `wget --limit-rate 1k https://docs.projectcalico.org/manifests/calico.yaml -O /tmp/calico.yaml` instaed of the -P option to `wget`

**Review by sudeeshjohn — APPROVED** [review]
/lgtm

**Comment by ltccci** [comment]
[APPROVALNOTIFIER] This PR is **NOT APPROVED**

This pull-request has been approved by: *<a href="https://github.com/ppc64le-cloud/k8s-ansible/pull/13#" title="Author self-approved">Rajalakshmi-Girish</a>*, *<a href="https://github.com/ppc64le-cloud/k8s-ansible/pull/13#pullrequestreview-653420578" title="LGTM">sudeeshjohn</a>*
To complete the [pull request process](https://git.k8s.io/community/contributors/guide/owners.md#the-code-review-process), please assign **mkumatag** after the PR has been reviewed.
You can assign the PR to them by writing `/assign @mkumatag` in a comment when ready.

The full list of commands accepted by this bot can be found [here](https://go.k8s.io/bot-commands?repo=ppc64le-cloud%2Fk8s-ansible).

<details open>
Needs approval from an approver in each of these files:

- **[OWNERS](https://github.com/ppc64le-cloud/k8s-ansible/blob/master/OWNERS)**

Approvers can indicate their approval by writing `/approve` in a comment
Approvers can cancel approval by writing `/approve cancel` in a comment
</details>

**Pull request merged**
